doc/userguide/src/CreatingTestData/Variables.rst

```diff
@@ -284,62 +284,67 @@ are imports, setups and teardowns where dictionaries can be used as arguments.
-`&{var}[item]` with dictionaries. The old syntax was deprecated in
+It is possible to access items of subscriptable variables, e.g. lists and
+dictionaries, using special syntax `${var}[item]`. Accessing items is an old
+feature, but prior to Robot Framework 3.1 the syntax was `@{var}[item]` with
+lists and `&{var}[item]` with dictionaries. The old syntax was deprecated in
 Robot Framework 3.2 and will not be supported in the future.
 
-Accessing list items
-''''''''''''''''''''
+.. _sequence items:
+
+Accessing sequence items
+''''''''''''''''''''''''
 
-It is possible to access a certain item of a list variable with the syntax
-`${var}[index]`, where `index` is the index of the selected value. Indices
-start from zero, negative indices can be used to access items from the end,
-and trying to access an item with too large an index causes an error.
-Indices are automatically converted to integers, and it is also possible to
-use variables as indices. List items accessed in this manner can be used
-similarly as scalar variables.
+It is possible to access a certain item of a `sequence`__ variable (e.g. list,
+string and bytes) with the syntax `${var}[index]`, where `index` is the index of
+the selected value. Indices start from zero, negative indices can be used to
+access items from the end, and trying to access an item with too large an index
+causes an error. Indices are automatically converted to integers, and it is also
+possible to use variables as indices. Sequence items accessed in this manner can
+be used similarly as scalar variables.
 
 .. sourcecode:: robotframework
 
    *** Test Cases ***
-   List variable item
+   Sequence variable item
        Login    ${USER}[0]    ${USER}[1]
        Title Should Be    Welcome ${USER}[0]!
 
    Negative index
-       Log    ${LIST}[-1]
+       Log    ${SEQUENCE}[-1]
 
    Index defined as variable
-       Log    ${LIST}[${INDEX}]
+       Log    ${SEQUENCE}[${INDEX}]
 
-List item access supports also the `same "slice" functionality as Python`__
+Sequence item access supports also the `same "slice" functionality as Python`__
 with syntax like `${var}[1:]`. With this syntax you do not get a single
-item but a slice of the original list. Same way as with Python you can
+item but a slice of the original sequence. Same way as with Python you can
 specify the start index, the end index, and the step:
 
 .. sourcecode:: robotframework
 
    *** Test Cases ***
    Start index
-       Keyword    ${LIST}[1:]
+       Keyword    ${SEQUENCE}[1:]
 
    End index
-       Keyword    ${LIST}[:4]
+       Keyword    ${SEQUENCE}[:4]
 
    Start and end
-       Keyword    ${LIST}[2:-1]
+       Keyword    ${SEQUENCE}[2:-1]
 
    Step
-       Keyword    ${LIST}[::2]
-       Keyword    ${LIST}[1:-1:10]
+       Keyword    ${SEQUENCE}[::2]
+       Keyword    ${SEQUENCE}[1:-1:10]
 
 .. note:: The slice syntax is new in Robot Framework 3.1 and does not work
           with the old `@{var}[index]` syntax.
```

Comment:
Need a note here about accessing items from strings/bytes/others being new in RF 3.2.